src/actions/player/playerHistogramActions.js

```diff
@@ -0,0 +1,66 @@
+import fetch from 'isomorphic-fetch';
+import { API_HOST } from 'config';
+import { playerHistogram } from 'reducers';
+import { getUrl } from 'actions/utility';
+import { bucketizeColumns } from 'utility';
+
+// const excludedOptions = ['limit'];
+const url =
```

Comment:
getUrl expects a function to generate the url, so this carrying allows that to still work.